NotificationBox/background.js

```diff
@@ -6,9 +6,8 @@ messenger.NotificationBox.onButtonClicked.addListener((windowId, notificationId,
   if (["btn-keep"].includes(buttonId)) {
     console.log("Box will not close, as long as one listener returns {close:false}.");
     return { close: false };
-  } 
+  }
     return { close: true };
```

Comment:
Strange that the indent here is not fixed by the linter. It is a left-over from the `else` being removed. Any idea?